### ISSUE TITLE
Uninstall 

### ISSUE THREAD

**Issue by brostosjoined** [bombsquad-community/plugin-manager#182]
So is it possible to like delete the files that a mod installs
( for its dependencies) when i uninstall the plugin.

**Comment by kingsamurai123** [comment]
Hello Brostosjoined,

The modules deletion while uninstalling the plugin is not recommended as there might be people use these modules outside of the game requirement.

**Comment by brostosjoined** [comment]
Yeah as you said that true but only on the monolithic build  of the game which will only be used on few specific platforms. I mean delete the modules, files and other junk that the mod install not the system modules
